docs/notebooks/gensim Quick Start.ipynb

```diff
@@ -51,17 +47,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-     "This is a particularly small example of a corpus for illustration purposes. Another example could be a list of all the plays written by Shakespeare, list of all wikipedia articles, or all tweets by a particular person of interest.\n",
+    "This is a particularly small example of a corpus for illustration purposes. Another example could be a list of all the plays written by Shakespeare, list of all wikipedia articles, or all tweets by a particular person of interest.\n",
```

Comment:
Hi
I couldn't figure out how to remove it form my PR
so I just undid the change I had made